vobject/base.py

```diff
@@ -689,13 +700,13 @@ def __str__(self):
         if self.name:
             return "<{0}| {1}>".format(self.name, self.getSortedChildren())
         else:
-            return u'<*unnamed*| {0}>'.format(self.getSortedChildren())
+            return "<*unnamed*| {0}>".format(self.getSortedChildren())
```

Comment:
Python 2.x issue

Reply:
black did this, will check

Reply:
I think we'll need to use a Black version that supports a target of py27.  Which I _think_ means 21.12b0 (I've tried pinning that and it seems to work).

I haven't yet compared the output side-by-side with this PR.